### PR TITLE
OpenCoarrays: fix for case-sensitive MacOS

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Follow-up to: https://github.com/macports/macports-ports/pull/16075

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: science/OpenCoarrays/Portfile [hunk 1]
```diff
@@ -31,6 +31,13 @@ checksums           rmd160  3e94a9fe478d60e027730e596ffe1c384de5da77 \
                     sha256  c08717aea6ed5c68057f80957188a621b9862ad0e1460470e7ec82cdd84ae798 \
                     size    329486
 
+pre-patch   {
+    file rename ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.f90 \
+        ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90_
+    file rename ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90_ \
+        ${worksrcpath}/src/tests/integration/pde_solvers/coarrayHeatSimplified/main.F90
+}
```
Comment (mascguy) [inline science/OpenCoarrays/Portfile:39]: Ah, I didn't realize you were using `file rename`.

Try using `move` instead, which should eliminate the need for a two-step process:

https://guide.macports.org/#reference.tcl-extensions

Comment (barracuda156) [reply science/OpenCoarrays/Portfile:39]: @mascguy Gonna try that and update today in a couple of hours.

**Comment by barracuda156** [comment]
@mascguy Thanks, `move` seems to be working just as fine – and shorter code.

**Comment by barracuda156** [comment]
> > @mascguy Thanks, `move` seems to be working just as fine – and shorter code.
> 
> Can you ensure the associated ticket is closed on merge:
> 
> https://trac.macports.org/ticket/65843

Added! Thank you.

**Review by mascguy — APPROVED** [review]

**Pull request merged**